### PR TITLE
Jamaican Jerk Grilled Eggplant

### PR DESCRIPTION
**Pull request by Kaalaayaa**

### CODE REVIEW

File: recipes/Kaalaayaa.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="author" content="Kalaya Chatelain">
```
Comment (Milad-mosadegh) [inline recipes/Kaalaayaa.html:5]: Nice and DONE! 👍

File: recipes/Kaalaayaa.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+ <img src="/img/JamaicanJerkGrilledEggplant.jpg" width="500px" alt="Jamaican Jerk Grilled Eggplant">
+
+<div>
+
+ <h1>Jamaican Jerk Grilled Eggplant</h1>
+
+ <h2>Ingredients</h2>
+
+ <ul>
```
Comment (Req) [inline recipes/Kaalaayaa.html:21]: The `ul` and the `ol` should really only contain `li` elements, meaning that the headings should be outside of them. 

Good job in any case! The recipe sounds great, I should really get some liquid aminos :thinking: -- also where can I get this Annies BBQ sauce?

Comment (Milad-mosadegh) [inline recipes/Kaalaayaa.html:21]: should move after line 23

File: recipes/Kaalaayaa.html [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    <li>4 cloves garlic, minced (~2 Tbsp minced garlic per 4 cloves)</li>
+    <li>1 Tbsp fresh grated ginger</li>
+    <li>3 Tbsp lime juice</li>
+    <li>1/4 cup tamari or coconut aminos (or soy sauce if not GF)</li>
+    <li>2-3 Tbsp coconut sugar or maple syrup (plus more to taste)</li>
+    <li>2 Tbsp melted coconut oil (or grape seed or avocado oil // plus more for grilling)</li>
+    <li>3 stalks green onions or scallions (thinly sliced)</li>
+    <li>1 medium serrano or habanero pepper (thinly sliced // seeds removed)</li>
+    <li>1 large eggplants (or sub 2 small per 1 large)</li>
+
```
Comment (Milad-mosadegh) [inline recipes/Kaalaayaa.html:40]: ul tag should be close here

File: recipes/Kaalaayaa.html [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    <h3>SAUCE</h3>
+
```
Comment (Milad-mosadegh) [inline recipes/Kaalaayaa.html:42]: here u have to make a new ul

**Pull request merged**